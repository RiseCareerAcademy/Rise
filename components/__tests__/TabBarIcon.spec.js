import 'react-native';
import React from 'react';
import tabBar from '../TabBarIcon';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
    const props = {
        name: 'name',
        focused: 'focused',
    };
  const tree = renderer.create(<tabBar {...props}>Snapshot test!</tabBar>).toJSON();

  expect(tree).toMatchSnapshot();
});
