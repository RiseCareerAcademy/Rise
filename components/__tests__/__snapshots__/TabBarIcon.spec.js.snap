// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<tabBar
  focused="focused"
  name="name"
>
  Snapshot test!
</tabBar>
`;
